### PR TITLE
Add the funding details

### PR DESCRIPTION
**Pull request by aschempp**
Let's see if this can make an impact 😎 

https://github.com/sponsors/contao-association

https://contao.org/en/news/launching-github-sponsors.html
https://contao.org/de/news/github-sponsors-fuer-contao.html
https://contao.org/de/supporter.html

### CODE REVIEW

**Review by fritzmg — DISMISSED** [review]

**Review by bytehead — DISMISSED** [review]

**Review by ausi — DISMISSED** [review]

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+github: contao-association
+custom: "https://to.contao.org/supporter"
```
Comment (ausi) [inline .github/FUNDING.yml]: I guess this needs to be changed to?

```suggestion
custom: https://to.contao.org/supporter
```

Comment (aschempp) [reply .github/FUNDING.yml]: According to https://docs.github.com/en/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository

> Note: If a custom URL in an array includes :, you must wrap the URL in quotes. For example, "https://www.paypal.me/octocat".

Comment (ausi) [reply .github/FUNDING.yml]: The code style checker doesn’t seem to like that, maybe try single quotes?

**Review by fritzmg — DISMISSED** [review]

**Review by leofeyer — CHANGES_REQUESTED** [review]
As discussed with @aschempp, we want to create a funding page on contao.org and link to it in the `README.md`. Once the page exists, we can add a `FUNDING.yml` file, too.

**Comment by aschempp** [comment]
@Tastaturberuf is working on that page 😎

**Comment by aschempp** [comment]
Updated for our new funding page 😎

**Comment by aschempp** [comment]
@leofeyer any update on this?

**Comment by leofeyer** [comment]
The `FUNDING.yml` file is already added here: https://github.com/contao/.github

**Review by leofeyer — APPROVED** [review]

**Pull request merged**